### PR TITLE
Tweak gradle, make code cleaner, don't track position, add MainActivity

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: app/src/main/java/mainburg/planetenweg/LocationMarker.java [hunk 1]
```diff
@@ -136,14 +164,16 @@ public void onLocationChanged(Location location) {
                 .title(activity.getResources().getString(R.string.your_location)));
         position.setIcon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN));
 
-        map.moveCamera(CameraUpdateFactory.newLatLngZoom(loc, map.getCameraPosition().zoom));
+        if (trackLocation) {
```
Comment (torkleyy) [inline app/src/main/java/mainburg/planetenweg/LocationMarker.java:167]: We  should probably allow to lock the view on this location by clicking on it, but I couldn't be bothered to read the API docs for the `GoogleMap`.

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    android:layout_height="match_parent"
+    android:layout_width="match_parent"
+    android:orientation="vertical">
+
+    <Button
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/show_map" />
+
+
```
Comment (torkleyy) [inline app/src/main/res/layout/activity_main.xml:12]: Yeah, extra whitespace 'cause I dunno what to put here.

Comment (Jolly5) [reply app/src/main/res/layout/activity_main.xml:12]: I don't know what we could put there

**Comment by torkleyy** [comment]
Any other concerns?

File: app/src/main/java/mainburg/planetenweg/MapActivity.java [hunk 3]
```diff
@@ -69,23 +64,42 @@ public void onMapReady(GoogleMap googleMap) {
...
+                m.setIcon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory
+                        .HUE_VIOLET));
             }
 
-            new PathRequest(waypoints, mMap, this).start();
+            runOnUiThread(new Runnable() {
+                @Override
+                public void run() {
+                    try {
+                        Drawing.drawPath(JsonData.getData(), googleMap);
```
Comment (Jolly5) [inline app/src/main/java/mainburg/planetenweg/MapActivity.java:79]: What is _JsonData.getData()_? What does this method do?

**Review by Jolly5 — APPROVED** [review]
Nice style updates. I am not really into code style but I assume you know exactly what you are doing.
I like your rework of the class PathRequest (well ok you deleted it and created a new class Drawing but you know what I mean)

**Comment by torkleyy** [comment]
Thanks for taking a look.

**Pull request merged**